templates/4a_compensate_image_data.ipynb

```diff
@@ -14,7 +14,7 @@
    "id": "237267d0",
```

Comment:
<p>Might be helpful to just also list the different types of sources we are compensating under spurious signal e.g. - organics, isotopic, Au... etc  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
will be added to readme